### DISCUSSION TITLE
History not working with Algolia I think?

### DISCUSSION THREAD

**Discussion by vonkoff** [epicweb-dev/epic-stack#261] [Q&A] [answered]
```shell
GET /?test_guj[menu][employmentType]=Temporary 200 - - 6.766 ms
ErrorResponse {
  status: 404,
  statusText: 'Not Found',
  internal: true,
  data: 'Error: No route matches URL "/build/_assets/font-IXXPCKZG.css.map"',
  error: Error: No route matches URL "/build/_assets/font-IXXPCKZG.css.map"
      at getInternalRouterError (/Users/vonkoff/GoodUnionJobs/remix/node_modules/@remix-run/router/dist/router.cjs.js:11:82478)
      at Object.query (/Users/vonkoff/GoodUnionJobs/remix/node_modules/@remix-run/router/dist/router.cjs.js:11:58604)
      at handleDocumentRequestRR (/Users/vonkoff/GoodUnionJobs/remix/node_modules/@remix-run/server-runtime/dist/server.js:11:4509)
      at requestHandler (/Users/vonkoff/GoodUnionJobs/remix/node_modules/@remix-run/server-runtime/dist/server.js:11:1850)
      at /Users/vonkoff/GoodUnionJobs/remix/node_modules/@remix-run/express/dist/server.js:11:426
}
No routes matched location "/build/_assets/font-IXXPCKZG.css.map" 
GET /build/_assets/font-IXXPCKZG.css.map 404 - - 5.074 ms
```

I am trying to use Algolia. My app was working before... I migrated over to epic stack. I am not sure what I have to change. I guess it is something somewhere where history of the window is change to go to the build because I am getting the error that "No route matches URL..." of the build. I don't remember every seeing this error in my old remix app.

**Comment by kentcdodds** [comment]
Hi @vonkoff,

Sorry, I'm confused about the sequence of events. So you migrated to epic stack and it was working, then you added algolia and it stopped working? Or is it that you were using algolia and it was working, then you migrated to epic stack and it stopped working?

Are the logs from production? It appears that someone's requesting sourcemaps for the font css file and they don't exist. That doesn't seem to be related to algolia.

**Reply by vonkoff** [reply, marked as answer]
Hey Kent,

Thanks for always being on the case...turns out it was a CSP issues. Had to add algolia.net as an exception. Actual results are loading in!->

```ts
app.use(
	helmet({
		crossOriginEmbedderPolicy: false,
		contentSecurityPolicy: {
			directives: {
				'connect-src': [
					MODE === 'development' ? 'ws:' : null,
					process.env.SENTRY_DSN ? '*.ingest.sentry.io' : null,
					"'self'",
					// THE PART BELOW :)
					'*.algolia.net', 
				].filter(Boolean),
// ...
```
